igrica.js

```diff
@@ -0,0 +1,62 @@
+let okvir = document.getElementById('okvir');
+let plava_kocka = document.getElementById('kocka');
+let zelena_kocka = document.getElementById('kocka-zelena');
+let crvena_kocka = document.getElementById('kocka-crvena');
+
+let frame = okvir.getBoundingClientRect();
+let blue = plava_kocka.getBoundingClientRect();
+let green = zelena_kocka.getBoundingClientRect();
+let red = crvena_kocka.getBoundingClientRect();
+
+let plavaY = 0;
+let plavaX = 0;
+
+plava_kocka.style.top = plavaY + "px";
+plava_kocka.style.left = plavaX + "px";
+
+
+
+
+///////////    FUNKCIJE     ////////////
+
+function destroyRed(){  // Crvena kocka nestaje pri dodiru sa plavom
+    if (plavaX + frame.left + 50 > red.left && plavaY + frame.top + 50 > red.top && plavaX + frame.left < red.left + 50 && plavaY + frame.top < red.top + 50){
```

Comment:
Da li mozes da objasnis zasto stoji ovo 50, tj koja je to vrednost? Mislim da treba izbegavati te hard kodirane numericke vrednosti. Ukoliko je to npr sirina kocke onda bi trebalo da prvo definises varijablu npr. let plava_width = 50 pa onda u 23em redu da bude umesto 50 if (plavaX + frame.left + plava_width)

Reply:
50 je sirina kocke. Dole u glue funkciji stoji 60 u if petlji jer nece da mi radi lepo sa 50. Zasto nece, to nemam pojma. Bice to ispravljeno.

Reply:
Da, to sto nece da radi na 50 a radi na 60 mi deluje kao jos jedna od velikih misterija javascript jezika. Provalio sam da se u mnogim situacijama js uopste ne ponasa kao neki drugi jezici sa kojima sam radio